artifact/artifact.sh

```diff
@@ -1,48 +1,89 @@
 #!/bin/bash -eux
-#
-# This code is meant to be executed within a TravisCI build.
+
+# This code is meant to be executed within a CI build.
 
 publish_to_dockerhub() {
-    if [ -n "${DOCKER_USERNAME:-}" ] && [ -n "${DOCKER_PASSWORD:-}"  ]; then
-        DOCKER_REPO="tidepool/${TRAVIS_REPO_SLUG#*/}"
-        echo "${DOCKER_PASSWORD}" | docker login --username "${DOCKER_USERNAME}" --password-stdin
+    if [ -n "${DOCKER_USERNAME:-}" ] && [ -n "${DOCKER_PASSWORD:-}" ]; then
```

Comment:
```suggestion
[ -n ${DOCKER_USERNAME:?} ] && [ ${DOCKER_PASSWORD:?} ]
```
Optional: With this, we'd get an error message specific to which variable wasn't set, and we could remove the conditional, de-indenting the bulk of the file and making it all a tiny bit easier to read.